composer/models/huggingface.py

```diff
@@ -23,8 +24,15 @@
 from composer.models.base import ComposerModel
 from composer.utils import MissingConditionalImportError, dist, get_file, import_object, is_model_fsdp, safe_torch_load
 
+try:
+    from peft import PeftModel, get_peft_model
+    _peft_installed = True
+except:
+    _peft_installed = False
```

Comment:
Is underscore notation consistent with rest of repo? I personally prefer `is_peft_installed` but whatever is consistent is best

Reply:
will check, I was thinking underscore because this isn't really intended for use outside of this file, but no strong opinion.